docs/_data/product/button-groups.yaml

```diff
@@ -0,0 +1,7 @@
+classes:
+  - class: .s-button-group
+    applies: N/A
+    description: Base button group style.
+  - class: .s-button-group--container
```

Comment:
Instead of relying on `form` here, I've designed this to allow for any arbitrary wrapping element. But, is this class name stupid?

Reply:
Nah I think that's good, broad name.